doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -98,6 +119,31 @@ Ceph binaries must be built for your branch before you can use teuthology to run
 .. _the Chacra site: https://shaman.ceph.com/api/search/?status=ready&project=ceph
 
 
+Pushing to ceph-ci repository
+*****************************
+
+Follow these steps to push to ceph-ci repository. After pushing, a new build will 
+automatically be scheduled. 
+
+1. Add the ceph-ci repository as a remote to local clone of ceph repository:
+
+   .. prompt:: bash $
+
+       git remote add ceph-ci git@github.com:ceph/ceph-ci.git  
+
+       $ git remote -v  
+       origin   git@github.com:ceph/ceph.git (fetch)  
+       origin   git@github.com:ceph/ceph.git (push)  
+       ceph-ci  git@github.com:ceph/ceph-ci.git (fetch)  
+       ceph-ci  git@github.com:ceph/ceph-ci.git (push)  
+
+2. Push your branch with changes to 
```

Comment:
s/Push your branch with changes to/Push your branch upstream by running a command of the following form:/